cli/cli-client/src/main/resources/help/benchmark.adoc

```diff
@@ -0,0 +1,71 @@
+BENCHMARK(1)
+============
+:doctype: manpage
+
+
+NAME
+----
+benchmark - runs a performance benchmark against a cache.
+
+The benchmark can be performed both for HTTP and for Hot Rod: the mode is determined by the protocol of the supplied URI.
```

Comment:
@tristantarrant I'm curious about "the mode is determined by the protocol of the supplied URI." How does the protocol determine the mode? Are there some benchmark modes that do not apply for certain protocols?